lib/apiForSheets.js

```diff
@@ -165,7 +240,11 @@ module.exports.checkClaStatus = async function (contextData) {
 
   // Authorize a client with the loaded credentials.
   const oauthClient = module.exports.authorize(JSON.parse(clientSecret));
+  const oauthClientWIthOnBoardingSecret = module.exports.
+    authorize(JSON.parse(OnBoardingSheetId));
```

Comment:
This does not seem correct. The same `OAuth client` should work for both sheets.

Reply:
Yes @jameesjohn corrected it in latest commit.